tests/glassflow/integration_tests/client_test.py

```diff
@@ -0,0 +1,15 @@
+def test_get_pipeline_ok(client):
+    pipeline_id = "bdbbd7c4-6f13-4241-b0b6-da142893988d"
+
+    p = client.get_pipeline(pipeline_id="bdbbd7c4-6f13-4241-b0b6-da142893988d")
+    assert p.id == pipeline_id
+
+
+def test_list_pipelines_ok(client, creating_pipeline):
+    res = client.list_pipelines()
+
+    assert res.status_code == 200
+    assert res.content_type == "application/json"
+    assert res.total_amount >= 1
+    assert res.pipelines[-1]["id"] == creating_pipeline.id
+    assert res.pipelines[-1]["name"] == creating_pipeline.name
```

Comment:
I feel we need to change it to follow at least the general unit-tests principles.
There are three stages: Initialization, Testing and Cleaning up.

So, we could define a dedicated space where we create a pipeline for each test, perform some assertions, and delete it afterward.

But definitely relying on a specific pipeline_id is not a good idea in the long run.

Reply:
Honestly, I'd rather say to delete these tests because "no test is better than a bad test."

Reply:
@vkd fixed, now we create a pipeline and delete it after the test

Reply:
Awesome, thank you!🚀